src/config.ts

```diff
@@ -102,66 +104,72 @@ const esSearch = new ElasticSearchService(
 );
 const s3DataService = new S3DataService(dynamoDbDataService, fhirVersion, { enableMultiTenancy });
 
-export const getFhirConfig = async (): Promise<FhirConfig> => ({
-    configVersion: 1.0,
-    productInfo: {
-        orgName: 'Organization Name',
-    },
-    auth: {
-        authorization: await getAuthService(),
-        // Used in Capability Statement Generation only
-        strategy: {
-            service: 'SMART-on-FHIR',
-            oauthPolicy: {
-                authorizationEndpoint: `${patientPickerEndpoint}/authorize`,
-                tokenEndpoint: `${patientPickerEndpoint}/token`,
-                introspectionEndpoint: `${oAuth2ApiEndpoint}/introspect`,
-                revocationEndpoint: `${oAuth2ApiEndpoint}/revoke`,
-                capabilities: [
-                    'context-ehr-patient',
-                    'context-standalone-patient',
-                    'permission-patient',
-                    'permission-user',
-                ], // https://www.hl7.org/fhir/valueset-smart-capabilities.html
+export const getFhirConfig = async (): Promise<FhirConfig> => {
+    if (issuerEndpoint === defaultEndpoint) {
```

Comment:
I think if you moved this condition into a separate function, then we could avoid all of the indentation changes to the code below. The function could take a suffix to append to the endpoint. And could be called more than once since you are only going to fetch parameter once.
For example:
```
oauthPolicy: {
                    authorizationEndpoint: await getIssuerEndpoint('/v1/authorize'),
                    tokenEndpoint: await getIssuerEndpoint('/v1/token'),
```